docs/api/flink/Function.md

```diff
@@ -1087,6 +1087,20 @@ Example:
 SELECT ST_IsClosed(ST_GeomFromText('LINESTRING(0 0, 1 1, 1 0)'))
 ```
 
+## ST_IsCollection
+
+Introduction: Returns `TRUE` if the argument type is a geometry collection type.
+
+Format: `ST_IsCollection(geom: geometry)`
+
+Since: `v1.5.0`
+
+Example:
+
+```sql
```

Comment:
Please provide two examples of the output. One is `true` and another one is `false`